pages/dkp/kommander/2.3/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -9,14 +9,14 @@ beta: false
 
 This section describes how to upgrade your Kommander Management cluster and all Platform Applications to their supported versions in networked, air-gapped, and on-prem environments. To prevent compatibility issues, you must first upgrade Kommander on your Management Cluster before upgrading to DKP.
 
-<p class="message--note"><strong>NOTE: </strong>It is important you upgrade Kommander BEFORE upgrading the Kubernetes version (or Konvoy version for Managed Konvoy clusters) in attached clusters, due to the previous versions' incompatibility with 1.22.</p>
+<p class="message--note"><strong>NOTE: </strong>It is important you upgrade Kommander BEFORE upgrading the Kubernetes version (or Konvoy version for Managed Konvoy clusters) in attached clusters, due to the previous versions' incompatibility with the Kubernetes version included in <a href="../../release-notes/">this release</a>.</p>
```

Comment:
Is this true for 2.3?  I know Kommander 2.1 was NOT compatible with Kubernetes 1.22, but that was because there were several older API's removed in 1.22 (https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22)

I don't see any API deprecations/removals for K8S 1.23 so I suspect Kommander 2.2 works just fine on Kubernetes 1.23.

It's probably still a good idea to upgrade Kommander first, but I suspect you don't *have* to.

Reply:
@floridoo @mhrabovcin could you please provide some info? If Craig is right, I would delete this note here and in several other places. Thanks!

Reply:
Craig is right, we don't have that requirement in 2.3 version. This requirement was very specific to 2.2.

Reply:
```suggestion
```